aiofiles/__init__.py

```diff
@@ -1,6 +1,7 @@
 """Utilities for asyncio-friendly file handling."""
 from .threadpool import open
-
+from .aiotempfile import NamedAioTemporaryFile, AioTemporaryFile
+from .wrap import wrap
 __version__ = '0.3.2'
 
-__all__ = (open, )
+__all__ = (open, NamedAioTemporaryFile, AioTemporaryFile, wrap)
```

Comment:
Adding `wrap` to public API maybe makes sense (the final decision is up to @Tinche sure) but if the function is a part of public API -- it should be documented. Right now aiofiles has no sphinx documentation (but I love to see aiofiles docs on readthedocs.io) -- but at least README should be updated.